README.md

```diff
@@ -1,40 +1,111 @@
+## Dependencies
 
+This project is dependent on Jenkins Pipeline and
+Jenkins >= 2.46.3, though some earlier versions may work, they have not been
+tested.
 
-## Getting help
+## Installation
 
-tk
+These shared libraries may be accessed one of three ways:
+1. Add this repo to 'Global Pipeline Libraries' in the Jenkins UI.
+1. Include a `libraries` blocking in declarative pipeline syntax.
+1. Include this library in an `@Library` statement in a Pipeline script.
 
-Instruct users how to get help with this software; this might include links to an issue tracker, wiki, mailing list, etc.
+### Global Pipeline Libraries
 
-**Example**
+See [this article](https://jenkins.io/doc/book/pipeline/shared-libraries/#global-shared-libraries)
+about adding shared libraries via the Jenkins UI.
 
-If you have questions, concerns, bug reports, etc, please file an issue in this repository's Issue Tracker.
+![](https://jenkins.io/doc/book/resources/pipeline/add-global-pipeline-libraries.png)
 
-## Getting involved
+### Within Pipeline Script
+
+The declarative way to include libraries is the following.
+
+```
+libraries {
+    lib('github.com/cfpb/jenkins-shared-libraries')
+}
+```
+
+Since this repo is hosted on github.com, it's easy to include it in all
+script dynamically by adding it to the `@Library` block within a Pipeline script
+
+```
+@Library('github.com/cfpb/jenkins-pipeline-shared-libraries') _
+```
+
+## Usage
+
+Currently this repo has a `sendEmail` var that can be inovked after
+importing it. Once the entire share library has been imported, `sendEmail`
+is available at the top level. E.g.
+
+```
+pipeline {
+    agent { label "master" }
+
+    libraries {
+        lib('github.com/cfpb/jenkins-shared-libraries')
+    }
 
-tk
+    stages {
+        stage("Echostage") {
+            steps {
+               echo "foo"
+            }
+        }
+    }
+    post {
+        always {
+            script {
+                sendEmail(currentBuild, ['testemail@domain.tld'])
+            }
+        }
+    }
+}"""
+```
 
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
+## TODO
+Maybe because we no longer have the `triggers`, 'fixed' emails no
+longer have 'Fixed' in the subject line, but instead say 'Building'. It would
+be nice if we could figure out why this is, and then get the 'Fixed' label
+back in the subject line. Note however that emails about failing builds do
+still say 'Failure' and 'Still Failing' in the subject.
+
+## Getting help
+
+If you have any trouble working with this repo, please open a github issue.
 
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
+## Getting involved
+
+Adding libraries to this repo is welcome and encouraged. Should you have
+an idea for a useful library or an improvement for an existing one fork
```

Comment:
'to an'

Reply:
This is not clear. Can you provide more context?

Reply:
'improvement to an' is more idiomatic than 'improvement for an' IMO, but please go with the one that sounds better to you

Reply:
I'm content with 'for an'. Let's stick with it.